misc/snapshotter/config.toml

```diff
@@ -21,15 +21,15 @@ daemon_cpu_profile_duration_secs = 5
 pprof_address = ""
 
 [daemon]
+# Specify a configuration file for nydusd
+nydusd_config = "/etc/nydus/nydusd-config.fusedev.json"
 nydusd_path = "/usr/local/bin/nydusd"
 nydusimage_path = "/usr/local/bin/nydus-image"
 # fusedev or fscache
 fs_driver = "fusedev"
 # How to process when daemon dies: "none", "restart" or "failover"
 recover_policy = "restart"
-# Specify a configuration file for nydusd
-nydusd_config = "/etc/nydus/nydusd-config.fusedev.json"
-# The fuse or fscache IO working threads started by nydusd
+# Nydusd worker thread number to handle FUSE or fscache requests, [0-1024]
```

Comment:
May be `[1-1024]`.

Reply:
0 means default, let nydusd to choose worker thread number:)

Reply:
Ok! May be we should explain the zero value here.

Reply:
added a comment for it.